src/lib/__tests__/plugins.test.ts

```diff
@@ -108,22 +102,18 @@ describe('editor view', () => {
 
     view.dispatch(view.state.tr.setMeta('update', true)) // trigger plugins
 
-    const tableOfContentsSection = view.state.doc.child(2).child(0)
-    expect(
-      tableOfContentsSection.content.child(1).attrs.contents
-    ).toMatchSnapshot()
-
-    const sectionWithEmptyParagraphs = view.state.doc.child(2).child(4)
-    expect(sectionWithEmptyParagraphs.childCount).toBe(2)
+    const sectionWithEmptyParagraphs = view.state.doc.child(2).child(3)
+    expect(sectionWithEmptyParagraphs.attrs.id).toBe('MPSection:empty-para')
+    expect(sectionWithEmptyParagraphs.childCount).toBe(3)
```

Comment:
I wonder why has the childcount increased here?

Reply:
Good catch. It turns out the "paragraphs" plugins, which is supposed to merge empty paragraphs, was not running since the doc never changes, so it looks like the "toc" plugin indirectly triggered it when it updates the content.